core/app/models/spree/payment_method.rb

```diff
@@ -1,5 +1,13 @@
 module Spree
-  # An abstract class which is implemented most commonly as a `Spree::Gateway`.
+  # A base class which is used for implementing payment methods.
+  #
+  # See https://github.com/solidusio/solidus_gateway/ for
+  # offically supported payment method implementations.
+  #
+  # Uses STI (single table inheritance) to store all implemented payment methods
+  # in one table (+spree_payment_methods+).
+  #
+  # This class is not ment to be instantiated. Please create instances of concrete payment methods.
```

Comment:
This is excellent class documentation. Wow. 

There's a typo in here though: "ment" should read "meant".